README.md

```diff
@@ -0,0 +1,81 @@
+# Covid19 Statistics App
+
+COVID-19 Stats is a single-page app that delivers up-to-date information on COVID-19 in-country, continent, and global total data. The date input filter allows the user to retrieve data on any given date. React, Redux, and Tailwind CSS were used to create the app.
+
+## Screenshots
+
+![react-capston](https://user-images.githubusercontent.com/92645881/163482229-b957948c-4f76-4f29-9b7f-f79bd0ce0cd0.gif)
+
+
+## Built With
+
+- HTML5, CSS3
+- JavaScript ES6, React/Redux, Bootstrap
+- CLI, TDD, gh-pages, Gitflow
+
+## Live demo
+
+[live Demo](covidstatupdate.netlify.app)
```

Comment:
- [ ] Let's add the correct live link of your project, I think it's `https://covidstatupdate.netlify.app/`.